packages/core/test/util/tokenTransferInspector.test.ts

```diff
@@ -221,6 +223,8 @@ describe('txInspector', () => {
 
   const assetProvider = createMockAssetProvider(assetInfos);
 
+  const timeout = 6000 as Milliseconds;
```

Comment:
Missing timeout test for both inspectors

Reply:
✅